modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectLayoutLocalServiceImpl.java

```diff
@@ -480,11 +484,50 @@ private void _validate(
 			throw new DefaultObjectLayoutException(
 				"There can only be one default object layout");
 		}
+
```

Comment:
What do you think to break your logic in another method? It would be more clear to understand. You can call this method like _validateRequiredFields or something like that.

Reply:
I changed to _validateSingleDefaultObjectLayout and _validateRequiredFields.